lib/faker/default/university.rb

```diff
@@ -5,22 +5,67 @@ class University < Base
     flexible :university
 
     class << self
+      ##
+      # Produces a random university name
```

Comment:
Can you add a period at the end of the sentence?

```suggestion
      # Produces a random university name.
```

The same applies to the other sentences in this PR.